src/python/pants/option/global_options.py

```diff
@@ -487,7 +487,9 @@ def from_options(cls, options: OptionValueContainer) -> LocalStoreOptions:
 
 
 class BootstrapOptions:
-    """This are the set of options necessary to create a Scheduler. If an option is not consumed
+    """The set of options necessary to create a Scheduler.
+    
+    If an option is not consumed...
```

Comment:
Oh this isn't complete, I meant for you to copy and paste the original you had. Sorry!

Reply:
lol